pages/event.php

```diff
@@ -291,20 +317,20 @@ function event() {
                     if (!$player->hasItem($system->event_data['red_lantern_id'])) {
                         throw new RuntimeException("You do not have this item!");
                     }
-                    if ($player->items[$system->event_data['red_lantern_id']]->quantity < 500) {
+                    if ($player->items[$system->event_data['red_lantern_id']]->quantity < 100) {
```

Comment:
```suggestion
                    if ($player->items[$system->event_data['red_lantern_id']]->quantity < $shadow_essence_cost['red_lanterns']) {
```
We could centralize these costs a bit and make them easy to change